lib/postgrex/protocol.ex

```diff
@@ -1593,6 +1606,16 @@ defmodule Postgrex.Protocol do
     transaction_error(s, postgres)
   end
 
+  defp parse_describe_comment_msgs(query, comment, tail) when is_binary(comment) do
+    statement = query.statement <> "/*#{comment}*/"
```

Comment:
It seems we may no longer go wit hthis PR, but I wonder if this could be dealt as IO lists:

```suggestion
    statement = [query.statement, "/*", comment, "*/"]
```